CONTRIBUTING.md

```diff
@@ -77,6 +77,57 @@ git remote add upstream git@github.com:virtual-kubelet/virtual-kubelet.git
 ```
 3. Submit a pull request.
 
+
+## Submission and Review guidelines
+
+We welcome and appreciate everyone to submit and review changes. Here are some guidelines to follow for help ensure
+a successful contribution experience.
+
+Please note these are general guidelines, and while they are a good starting point, they are not specifically rules.
+If you have a question about something, feel free to ask.
```

Comment:
Ask where? ;-)

I would say #virtual-kubelet on Kubernetes Slack? I know we have a CNCF mailing list somewhere too..

Reply:
Added